packages/contracts/contracts/minter-suite/Minters/MinterSetPriceERC20V5.sol

```diff
@@ -492,6 +493,12 @@ contract MinterSetPriceERC20V5 is
             coreContract: coreContract
         });
 
+        // validate that the price sent is greater than or equal to the price per token
+        require(
+            purchasePricePerToken >= pricePerTokenInWei,
+            "Must send amount greater than or equal to price per token"
```

Comment:
Prefer to keep revert messages <= 31 characters (helps keep everything in single slot, reducing contract bytecode size)
```suggestion
            "Only max price gte token price"
```

Reply:
Makes sense, TY!